src/gui/search/searchsortmodel.h

```diff
@@ -49,8 +52,45 @@ class SearchSortModel: public QSortFilterProxyModel
 
     explicit SearchSortModel(QObject *parent = 0);
 
+    void enableNameFilter(bool enabled);
+    void setNameFilter(const QString& searchTerm = QString());
+
+    //! \brief setSizeFilter Sets parameters for filtering by size
```

Comment:
BTW, we can think about using doxygen for generating a .qch file for Qt Creator...